packages/storybook/src/stories/Alert.stories.tsx

```diff
@@ -107,7 +107,7 @@ Toast.args = {
   description: 'Description Copy goes here',
   variant: 'success',
   type: 'toast',
-  onClose,
+  onClose: undefined,
```

Comment:
Nit:
`Null is the intentional absence of a value (null is explicit)`
`Undefined is the unintentional absence of a value (undefined is implicit)`

So if we are initializing something it's better to use null, unless there is a reason why we should use undefined (if yes, let me know).

Reply:
I agree